source/Playnite.DesktopApp/ViewModels/InstalledGamesViewModel.cs

```diff
@@ -340,24 +340,35 @@ public void ConfirmDialog()
 
         public void SelectExecutable()
         {
-            var path = dialogs.SelectFile("Executable (.exe)|*.exe");
+            var path = dialogs.SelectFile("Executable (.exe,.bat,lnk)|*.exe;*.bat;*.lnk");
             if (string.IsNullOrEmpty(path))
             {
                 return;
             }
+            var program = Common.Programs.GetProgramData(path);
 
-            var productName = FileVersionInfo.GetVersionInfo(path).ProductName;
-            var import = new ImportableProgram(new Program()
+            // Ignore all target files that are not bat or exe files
+            if (!program.Path.EndsWith(".exe", StringComparison.OrdinalIgnoreCase) &&

```

Comment:
I don't think we should have this restriction. If somebody decides to import .lnk, then we should not care what that .lnk points to.

Reply:
resolved in https://github.com/felixkmh/Playnite/commit/8ac8f5f57d286b3a1a80bdd23bbe0e0df33ad885

Reply:
I think the rationale behind this was that you previously couldn't select anything but .exe and .bat files in the first place. The only reason you could select .lnk files was that the file selection dialogue returned the path to the file the shortcut pointed to, which then had to match the filter.